src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -69,23 +53,8 @@
   @Setter(lombok.AccessLevel.NONE)
   ExpandableField<Customer> customer;
 
-  /**
-   * ID of the default payment method for the subscription schedule. If not set, invoices will use
-   * the default payment method in the customer's invoice settings.
-   */
-  @SerializedName("default_payment_method")
-  @Getter(lombok.AccessLevel.NONE)
-  @Setter(lombok.AccessLevel.NONE)
-  ExpandableField<PaymentMethod> defaultPaymentMethod;
-
-  /**
-   * ID of the default payment source for the subscription schedule. If not set, defaults to the
-   * customer's default source.
-   */
-  @SerializedName("default_source")
-  @Getter(lombok.AccessLevel.NONE)
-  @Setter(lombok.AccessLevel.NONE)
-  ExpandableField<PaymentSource> defaultSource;
+  @SerializedName("default_settings")
+  DefaultSettings defaultSettings;
```

Comment:
I notice that a lot of fields in this file seem to have a docblock, in case you want to add one.

Reply:
yeah it's autogen-ed from the spec so not something I should add manually. But nice catch, I'll get the team to fix it